account-kit/smart-contracts/src/6900-v08-RI/account/account.ts

```diff
@@ -0,0 +1,121 @@
+import type {
+  EntryPointParameter,
+  SmartAccountSigner,
+  SmartContractAccountWithSigner,
+  ToSmartContractAccountParams,
+} from "@aa-sdk/core";
+import {
+  createBundlerClient,
+  getAccountAddress,
+  getEntryPoint,
+  toSmartContractAccount,
+} from "@aa-sdk/core";
+import {
+  concatHex,
+  encodeFunctionData,
+  type Address,
+  type Chain,
+  type Hex,
+  type Transport,
+} from "viem";
+import { AccountFactoryAbi } from "../abis/AccountFactory.js";
+import {
+  getDefaultRIAccountFactoryAddress,
+  DEFAULT_OWNER_ENTITY_ID,
+} from "../utils.js";
+import { executor } from "../actions/execute.js";
+import { singleSignerMessageSigner } from "../modules/single-signer-validation/signer.js";
+
+export type SingleOwnerRIAccount<
```

Comment:
Should we use `SingleOwner` wording in SDK? 
After multiple validations changes, `SingleSigner` seems fit better.

Reply:
Yes good call, will update